### ISSUE TITLE
My problem with the left stick.

### ISSUE THREAD

**Issue by Isrdvnt** [sam0x17/XJoy#61]
Hello, the last days i've been using Xjoy, but there's the problem, when i close xjoy, while i'm playing my games, the left stick is detected and move like left-foward, please help, how can i fix it.

**Comment by sam0x17** [comment]
So it's running/doing stuff after you close XJoy? Or are you saying there is a problem with the left stick while XJoy is running?

**Comment by Isrdvnt** [comment]
No, the problem is that when I play a game that is compatible with the xbox control, the game detects as if I were moving the stick to the left, I can send a video.

**Comment by popiomondayz** [comment]
so you mean...
_joycon drift?_
